.changeset/wicked-baboons-pull.md

```diff
@@ -0,0 +1,5 @@
+---
+'lit-html': patch
+---
+
+Add more detail to some hydration errors
```

Comment:
Changelog is wrong

Reply:
Oops. Did I put the wrong changeset on the other PR too?